README.MD

```diff
@@ -1,22 +1,40 @@
 # KeePass Secret Management Extension
+
+## Actions
```

Comment:
Normally I'd require stuff like this a separate PR but since it's just docs I'll be OK with it :)